index.html

```diff
@@ -1,262 +1,209 @@
-            <h4>Frameworks</h4>
-            <img src="./images/Union.png" alt="icon">
+    </section>
+    <section id="contact" class="contact-me">
+        <div class="bg-image">
+            <h1 class="contact-title">Contact me</h1>
+            <p class="contact-text">
+                If you have an application you are interested in developing, a feature
+                that you need built or a project that needs coding. I’d love to help
+                with it
+            </p>
+            <form action="https://formspree.io/f/xqkgeweb" method="POST" class="contact-form">
+                <input type="text" name="name" id="username" placeholder="Your Name" maxlength="30" required />
+                <input type="email" name="email" id="useremail" placeholder="Your Email" required />
+                <textarea name="message" id="usermessage" cols="30" rows="10" maxlength="500"
+                    placeholder="Write your message here" required></textarea>
+                <div class="button-resume btn-form">
+                    <button class="btn" type="submit">Get in touch</button>
+                </div>
+            </form>
         </div>
-      <hr>
-      <div class="languages_skills">
-        <h4>Skills</h4>
-        <img src="./images/Union.png" alt="icon">
-    </div>
-    <hr>
-</div>
-
-</section>
-<section id="contactlast_section">
-    <div class="last_section2">
-    <div class="contact_me">
-        <h1 >Contact me</h1>
-
-    </div>
-    <p class="last_para">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-    
-    <form class="contact_form_last_section" action="https://formspree.io/f/xbjwkevl" method="post">
-        
-   <input type="text" placeholder="Enter your name"  maxlength="30" required class="text1" name="name"> 
-
-   <input type="email" placeholder="Enter your Email" required class="text1" name="email">
- 
-   <textarea class="textarea_form" required rows="4" maxlength="500" placeholder="Write your message here" cols="50" name="message"></textarea>
+    </section>
 
+    <div class="modal-example">
+        <div class="modal-head">
+          <h1 class="projectname"></h1>
+          <button type="button" class="closeModal">&times;</button>
+        </div>
+        <div class="modal__body">
+          <div class="short_desc">
+            <h4 class="company"></h4>
+            <span></span>
+            <h4 class="role"></h4>
+            <span></span>
+            <h6 class="year"></h6>
+          </div>
+          <img src="#" alt="img" class="modal-img">
+          <div class="project-desc">
+            <p class="desc">Lorem ipsum dolor sit amet consectetur adipisicing elit. In voluptatem pariatur modi rerum error
+              ea consequatur, excepturi eum maiores! Quo quibusdam culpa ratione.?Lorem ipsum dolor sit amet, consectetur
+              adipisicing elit.
+              !</p>
```

Comment:
The projects are displayed dynamically from the array of objects. Kudos for this. 👍🏽 However, the project description for the pop-up window is being hardcoded from the HTML file.

- [ ] Make The project description in the pop-up window dynamic as well, just like the other sections.

Compare these images
![five](https://user-images.githubusercontent.com/65501535/163577998-1366f91a-7136-4594-b3a5-d750cdedbc22.png)

![four](https://user-images.githubusercontent.com/65501535/163578019-97a0fe43-8e31-4d34-9a65-8cd905e443ec.png)

Reply:
@mke2111 hi, kindly check the image on the Figma for the pop-up and compare it with the instruction you have given me. I have attached the picture below from Figma otherwise the pop up is dynamic using js you can check the code in the js file.
<img width="627" alt="Screenshot 2022-04-15 at 10 20 06 PM" src="https://user-images.githubusercontent.com/84595837/163598151-dda64e06-dc09-4222-89b4-809a634c3007.png">

Reply:
The logic in this should be that the project details for a project should be the same for the popup project details. Remember that the details for each will make each project implementation simpler.